bitcoin/_posts/0000-00-00-advent-27.md

```diff
@@ -0,0 +1,287 @@
+Every contributor has the right to review, and ACK or provide a reasoned NACK.
+Even if everyone else is excited about something doesn't mean there isn't space
+for new, thought through, dissent.
+1. We would need to merge the implementation. This is simple, but enough ACKs have to
+   come in and rebases on other subsequent changes to get it in. This can
+   happen ahead of 'full consensus' since there are no deployment parameters,
+   but aids in increasing the testing priority of CTV.
+1. We would need to get a majority of miners/pools primed to accept the upgrade.
+1. We would need to decide on release parameters for the implementation.
+1. We would need to merge and release a client with release parameters
+1. The client would needs to lock-in by a supermajority of miners signalling.
+1. Then, and only then, would CTV be fully available.
+
+### What's the Maximally Aggressive/Optimistic Timeline?
+
+1. Soft Signals / Developer Review: 2-3 months required to get ACKs on the
+implementation, assuming no major changes required.
+1. Merge: Instant, just whenever there are ACKs against the implementation being
+safe and matching BIP.
+1. Getting miners primed to signal: ~20% of pools are on
+[utxos.org/signals](https://utxos.org/signals), more should be coming on board
+soon. Don't expect this to take additional time if in conjunction with Developer
+review.
+1. Debating Timelines: ~1 month to agree on a release timeline
+1. Preparing a release: ~1 month to do release candidates and testing.
+1. Speedy Trial: 3 months of signalling, 3 months of waiting for lock-in to active.
+
+
+### Overall, it could look like this:
+
+On March 15th developers reach agreement on merging BIP-119's implementation.
+On April 15th, agreement is reached on release parameters for signalling from ~
+June 1st to ~September 1st. The active height would be November 10th.  A client
+is prepared, and tested, and released. No issues are found. The miners signal at
+some point in the 3 month window above the threshold. CTV locks-in. Developers
+can prep wallet software for deeper integration. CTV activates before
+Thanksgiving, avoiding the "dire straits" of
+thanksgiving-hanukkah-christmas-chinese-new-year-valentines-day season.
+
+_but that's basically identical to Taproot's timeline?_
+
+Exactly. If we act on this timeline starting in early January 2022, it is
+possible to meet an almost identical timeline for CTV as Taproot.
+
+Part of why it works is that the next major release is scheduled for 2022-04-01.
+Soft forks are usually released as a minor patch on top of a few recent major
+releases. So CTV could be:
+
+1. Code included in 23.0 for CTV, no activation parameters
+1. Activation parameters in 23.1, and backported to 22.x, 21.x, and (really this is up to maintainers how far to backport!).
+1. 23.1 released before June 1st, 2022.
+
+**What could go wrong?**
+
+1. **Concern:** There's a small tweak to CTV that makes it marginally better, and it's worth
```

Comment:
I'd phrase this as "there could be a small tweak". I read this initially as being that there are already is one under discussion.